### PR TITLE
Improved GitHub issues

### PR DESCRIPTION
**Pull request by damienwebdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
+-->
+
+# Feature request
+
+## Feature Name
+<!-- Naming is hard, what YOU call this feature? -->
+
+## The desired behavior
```
Comment (xelaint) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Stay consistent with `expected`?

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+<!--
+PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+
+ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
+-->
+
+# Documentation Request
+
+## Prior Works
```
[the remark this answers is not in the document]
Comment (damienwebdev) [reply .github/ISSUE_TEMPLATE/documentation.md]: @lderrickable I clarified what I meant.

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+<!-- Describe any existing documentation that this would potentially require change. -->
```
[the remark this answers is not in the document]
Comment (damienwebdev) [reply .github/ISSUE_TEMPLATE/documentation.md]: @lderrickable fixed.

File: .github/ISSUE_TEMPLATE/documentation.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## Current behavior
+<!-- Describe how to came to need the documentation. -->
```
[the remark this answers is not in the document]
Comment (damienwebdev) [reply .github/ISSUE_TEMPLATE/documentation.md]: fixed.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+<!--
+PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+
+ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
+-->
+
+# Feature request
+
+## Feature Name
+<!-- Naming is hard, what YOU call this feature? -->
```
[the remark this answers is not in the document]
Comment (damienwebdev) [reply .github/ISSUE_TEMPLATE/feature_request.md]: fixed.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+## The desired behavior
+<!-- Please describe, in as much detail as you can, what you'd like to see happen. -->
+
+## Your Use Case
+<!-- Please try to format as "As a {role}, I'd like to be able to do {x}. -->
+
+## Prior Works
```
[the remark this answers is not in the document]
Comment (damienwebdev) [reply .github/ISSUE_TEMPLATE/feature_request.md]: fixed

[the remark this answers is not in the document]
Comment (damienwebdev) [reply .github/ISSUE_TEMPLATE/feature_request.md]: fixed

File: .github/ISSUE_TEMPLATE/performance_issue.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+-->
+
+# Performance Issue
+
+## Current behavior
+<!-- Describe how the issue manifests. -->
+
+
+## Expected behavior
+<!-- Describe what the desired behavior would be. -->
```
[the remark this answers is not in the document]
Comment (damienwebdev) [reply .github/ISSUE_TEMPLATE/performance_issue.md]: fixed

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (damienwebdev) [reply .github/ISSUE_TEMPLATE/feature_request.md]: fixed

**Comment by damienwebdev** [comment]
@lderrickable needs re-reviewed.

File: .github/ISSUE_TEMPLATE/performance_issue.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (xelaint) [reply .github/ISSUE_TEMPLATE/performance_issue.md]: `<!-- Describe what the desired behavior would be. -->`

Still says desired?

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (xelaint) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Still says desired

File: .github/ISSUE_TEMPLATE/performance_issue.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (damienwebdev) [reply .github/ISSUE_TEMPLATE/performance_issue.md]: woops, forgot to push.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (damienwebdev) [reply .github/ISSUE_TEMPLATE/feature_request.md]: fixed.

**Comment by xelaint** [comment]
lgtm

**Pull request merged**